arviz/plots/violinplot.py

```diff
@@ -61,6 +62,10 @@ def plot_violin(
         intervals. Defaults to ``True``.
     rug: bool
         If ``True`` adds a jittered rugplot. Defaults to ``False``.
+    side : str, default "both"
```

Comment:
As this can only take a fixed set of values, I would update this to

```suggestion
    side : {"both", "left", "right"}, default "both"
```

I would also add a check before calling the backend function + rise an error if the value passed doesn't match either of this. Otherwise the result might be strange or end up running into hard to diagnose errors.